src/backups.py

```diff
@@ -149,14 +195,61 @@ def _on_create_backup_action(self, event) -> None:
             event.fail(validation_message)
             return
 
+        # Retrieve the S3 Parameters to use when uploading the backup logs to S3.
+        s3_parameters, _ = self._retrieve_s3_parameters()
```

Comment:
thoughts on uploading a metadata file to S3 before running `pgbackrest backup`. this will surface bucket permissions issues before running a backup (which could be time consuming)